### PR TITLE
Chroma Siting on Linux

### PR DESCRIPTION
**Pull request by fzhar**

### CODE REVIEW

File: _studio/shared/src/mfx_vpp_vaapi.cpp [hunk 1]
```diff
@@ -1237,6 +1243,41 @@ mfxStatus VAAPIVideoProcessing::Execute(mfxExecuteParams *pParams)
         break;
     }
 
+#if (MFX_VERSION >= 1025)
+        uint8_t& chromaSitingMode = m_pipelineParam[0].input_color_properties.chroma_sample_location;
+        chromaSitingMode = VA_CHROMA_SITING_UNKNOWN;
```
Comment (dmitryermilov) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: should be in switch, I suppose

Comment (dmitryermilov) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: should be in switch, I suppose

Comment (dmitryermilov) [reply _studio/shared/src/mfx_vpp_vaapi.cpp]: Also you should add a reference to https://github.com/Intel-Media-SDK/MediaSDK/issues/1737 via:
Fixes: # 1737

Comment (fzhar) [reply _studio/shared/src/mfx_vpp_vaapi.cpp]: It can be left here - this is just local reference, it will be destroyed after exiting function scope.

Comment (fzhar) [reply _studio/shared/src/mfx_vpp_vaapi.cpp]: "Fixes" issue is addressed

**Pull request merged**